.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+@holgiosalos @renardete @leonleo997 @valeryibarra
```

Comment:
Add a line break because the end of file is missing

Reply:
Do not forget the `*`  at the beginning 
![image](https://user-images.githubusercontent.com/35767777/88468745-a3bda400-cead-11ea-9f7a-199a3dfb92dc.png)

Reply:
Done